src/Core/Framework/Api/ApiDefinition/Generator/EntitySchemaGenerator.php

```diff
@@ -89,6 +91,8 @@ private function getEntitySchema(EntityDefinition $definition): array
         return [
             'entity' => $definition->getEntityName(),
             'properties' => $properties,
+            'write-protected' => $definition->getProtections()->get(WriteProtection::class) !== null,
```

Comment:
When both protections are active. Should be not rather exclude the complete entity? The user cannot interact with it

Reply:
yep skip it